commons/src/main/java/esa/commons/StringUtils.java

```diff
@@ -204,7 +204,9 @@ public static String concat(String... params) {
         if (params.length < 5) {
             String ret = params[0];
             for (int i = 1; i < params.length; i++) {
-                ret += params[i];
+                if (params[i] != null) {
+                    ret += params[i];
+                }
             }
```

Comment:
What if the params[0] is `null`.
Maybe it would be better if we just remove this logic(204~212)